region/crateria/central/Climb.json

```diff
@@ -881,6 +922,48 @@
         "Enemies will not hurt Samus since they are non-global and also off-camera."
       ]
     },
+    {
+      "link": [2, 5],
+      "name": "Zebes Ablaze - R-Mode Spark Interrupt (Gain Blue Suit)",
+      "entranceCondition": {
+        "comeInWithRMode": {}
+      },
+      "requires": [
+        "Gravity",
```

Comment:
The enemyDamage event is a little more important than a way to kill them.  But you need way past the pirates before you can CF.

Reply:
What I added will assume taking a hit from the pirates before CF if you don't have some weapons that can kill them fast (supers, any kind of Plasma, or power bomb). Charge+IWS can also quickly kill one but has to recharge to kill the other, which may be too much on top of the extra shot cooldown due to X-Ray.